app/components/dummy-input/dummy-input.tsx

```diff
@@ -43,13 +43,18 @@ export interface DummyInputProps extends PressableProps {
    * The value to display
    */
   value?: string
+
+  /**
+   * Pass a component to present at the other side of the dummy input
+   */
+  sideSection?: React.ReactNode
```

Comment:
```suggestion
  endNode?: React.ReactNode
```
I think it's a better name